docs/pom.xml

```diff
@@ -6,47 +6,53 @@
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>spring-cloud-etcd</artifactId>
-		<version>1.0.0.BUILD-SNAPSHOT</version>
+		<version>1.0.1.BUILD-SNAPSHOT</version>
```

Comment:
This isn't a patch update, so update to version 2.0.0-SNAPSHOT

Reply:
Fixed